CartApi/Infrastructure/Filters/HttpGlobalExceptionFilter.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+using ShoesOnContainers.Services.CartApi.Infrastructure.ActionResults;
```

Comment:
once namespace names are changed, update here to include the appropriate namespace. Lines 2, 3 and line 10